tests/ecosystem/upgrade/test_upgrade.py

```diff
@@ -52,3 +54,25 @@ def test_upgrade():
     """
 
     run_ocs_upgrade()
+
+
+@post_ocs_upgrade
+@pytest.mark.polarion_id("OCS-2449")
+def test_check_mon_pdb_post_upgrade():
+    """
+    Function to check disruptions_allowed and minimum
+    available mon count
+
+    """
+    ceph_obj = CephCluster()
+
+    # Check for mon count
+    mons_after_upgrade = ceph_obj.get_mons_from_cluster()
+    log.info(f"Mons after upgrade {mons_after_upgrade}")
+
+    disruptions_allowed, min_available_mon = get_mon_pdb()
+    log.info(f"Number of Mons Disruptions_allowed {disruptions_allowed}")
+    log.info(f"Minimum_available mon count {min_available_mon}")
+
+    assert disruptions_allowed == 1, "Mon Disruptions_allowed count not matching"
+    assert min_available_mon == 2, "Minimum available mon count is not matching"
```

Comment:
the mon pdb values got changed from 4.5 on wards, may be add a comment about it just for info purpose

Reply:
Done